simulator/src/main/java/com/hazelcast/simulator/utils/PropertyBindingSupport.java

```diff
@@ -178,6 +164,19 @@ private static Field findPropertyField(Class clazz, String property) {
         }
     }
 
+    private static boolean setValue(Object object, String value, Field field) throws IllegalAccessException {
```

Comment:
overall the commit is very good, but this method might be misleading - it doesn't set anything if the field belongs to a probe.